.github/ISSUE_TEMPLATE/report_a_bug.yml

```diff
@@ -1,18 +1,8 @@
 name: 🐛 Report a bug
-description: Have you found a bug or issue? Create a bug report for OpenFGA
+description: If you are reporting that Check, ListObjects or ListUsers API are returning unexpected responses, please report a security vulnerability instead
```

Comment:
We may need to qualify this by saying there is a Check, ListObject, ListUser behaviour delta between version. Otherwise, I am concerned that end user misunderstanding may trigger this flow.

Reply:
I wouldn't like to put the burden on the reporter to confirm whether the issue can be reproduced in another version, we can do that for them.